CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 ### Internal
 
 - Upgrade testbed and convenience api folder to Plone 6 final @sneridagh [#4105](https://github.com/plone/volto/issues/4105)
+- Add HI-ERN website to "Volto in Production" section in README @steffenri
```

Comment:
@steffenri volto now uses towncrier for release notes. you need to add a file in the news folder

Reply:
See https://6.docs.plone.org/volto/developer-guidelines/contributing.html#create-a-pull-request

Reply:
Fixed. But I wonder how I can know the PR number before even creating the PR ;)

Reply:
@steffenri I usually create the PR, then add a new commit to the branch with the changelog.

Reply:
> Fixed. But I wonder how I can know the PR number before even creating the PR ;)

Volto developers have the ability to see into the future. It will come to you. :grin: